### PR TITLE
[lldb] Fix log messages in resolveRemoteAddress

### PR DESCRIPTION
**Pull request by kastiglione**

### CODE REVIEW

**Review by augusto2112 — APPROVED** [review]
Whoops! Thanks!

**Comment by kastiglione** [comment]
@swift-ci test

**Comment by kastiglione** [comment]
@swift-ci test Linux

**Comment by kastiglione** [comment]
@swift-ci test

**Comment by kastiglione** [comment]
@swift-ci test

File: lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp [hunk 1]
```diff
@@ -329,16 +329,16 @@ LLDBMemoryReader::resolveRemoteAddress(uint64_t address) const {
   Address resolved(file_address, object_file->GetSectionList());
   if (!resolved.IsValid()) {
     LLDB_LOG(log,
-             "[MemoryReader] Could not make a real address out of file "
-             "address {1:x} and object file {}",
+             "[MemoryReader] Could not make a real address out of file address "
+             "{0:x} and object file {}",
```
Comment (augusto2112) [inline lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp]: Just tested this, `{}` will crash, change it to `{1}`

Comment (kastiglione) [reply lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp]: done

Comment (kastiglione) [reply lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp]: The docs appear to indicate the index isn't required

https://github.com/llvm/llvm-project/blob/287ce6b51675aee43870bebfff68bb144d1ab90e/llvm/include/llvm/Support/FormatVariadic.h#L175

**Comment by kastiglione** [comment]
@swift-ci test

File: lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp (hunk 1 above)
Comment (augusto2112) [reply lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp]: Weird, maybe it's a bug then? Or maybe if you specify one position you have to specify all of them?

**Review by augusto2112 — APPROVED** [review]

File: lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp (hunk 1 above)
Comment (kastiglione) [reply lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp]: I wondered the same, in any case I'll stick with explicit indexes. None of the unit tests for formatv seem to use `{}`, so an index must be required.

Comment (kastiglione) [reply lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp]: https://reviews.llvm.org/D118833

**Pull request merged**